ex03.exs

```diff
@@ -50,13 +50,27 @@ defmodule Ex03 do
      it is nicely structured            7
   """
 
-  def pmap(collection, process_count, function) do
-    # your code goes here
-    # I'm hoping to see a simple pipeline as the body of this function...
+# "running" function
+def pmap(collection, process_count, function) do
+    Enum.chunk_every(collection, div(Enum.count(collection),process_count))
```

Comment:
`div` returns the wrong value for the chunk size (try it with 10 and 4: you get 2, but it should be 3)

# -2